app/src/NativeBase/Components/Forms/DatePicker.tsx

```diff
@@ -31,10 +31,14 @@ const DatePicker: FC<DatePickerProps> = ({
   const { colorMode } = useColorMode()
   const { width } = Dimensions.get('window')
 
+  const showDatePicker = () => {
```

Comment:
Small thing, but as technically this toggles between showing/hiding the date picker, could it be named something like `showHideDatePicker` or `toggleShowDatePicker`?

Reply:
This has been updated - thanks Joe!